HATE/Shuffle.cs

```diff
@@ -405,19 +404,18 @@ public static bool HitboxFix_Shuffler(UndertaleChunk _chunk, UndertaleData data,
                 }
             }
             UndertaleSprite.MaskEntry newEntry = new UndertaleSprite.MaskEntry();
-            int numBytes;
-            numBytes = maskingBitArray.Length / 8;
+            int numBytes = maskingBitArray.Length / 8;
```

Comment:
Just saying that I rember that that was what ImportGraphics.csx did

Reply:
yeah but its the same internally.